recipe/meta.yaml

```diff
@@ -34,18 +36,29 @@ requirements:
     - toolchain
     - boost-cpp 1.63.*
     - cmake
-    - zlib 1.2.*
-    - snappy
-    - brotli
-    - thrift-cpp
+    - zlib 1.2.*        # [unix]
+    - snappy            # [unix]
+    - brotli            # [unix]
+    - thrift-cpp        # [unix]
+    - python            # [win]
```

Comment:
@xhochy , cmake script of parquet-cpp doesn't work properly yet on Windows with 3rd party libs defined via *_HOME directories. It seems only boost can be fined fine on Windows. I've limited installation of all libs only to unix-like systems, because on Windows they are not used and potentially can introduce undesired build failures in future when cmake script of parquet-cpp will be updated.  So, the list of dependencies is the same for all platforms, but on unix-like system the 3rd party libs are taken from conda-forge and on windows they are built from sources.